games/xworld3d/xworld3d.cpp

```diff
@@ -137,7 +137,7 @@ X3World::X3World(const std::string& conf, bool print_conf, bool big_screen) :
     }
 
     world_ = util::make_unique<roboschool::World>(
-            FLAGS_x3_gravity, FLAGS_x3_time_step);
+            img_height_, img_width_, FLAGS_x3_gravity, FLAGS_x3_time_step);
```

Comment:
Why height and width are needed?

Reply:
it was hard coded in roboschool. i changed it.